packages/prosess-aarskvantum-oms/src/components/Årskvantum.tsx

```diff
@@ -46,17 +46,25 @@ const InfoRammemelding = styled.span`
   }
 `;
 
-export const konverterDesimalTilDagerOgTimer = (desimal: number) => {
+export interface DagerTimer {
+  dager: number;
+  timer?: number;
+}
+
+const formaterTimerDesimal = (timerDesimal: number): number =>
+  Number.parseFloat(timerDesimal.toFixed(2).replace('.0', '').replace(/\.*0/, ''));
```

Comment:
Denne såg litt pussig ut. Kvifor to replace?

Reply:
Sant, jeg glemte å fjerne den første delen (den andre matcher også den første)